README.md

```diff
@@ -27,7 +27,7 @@
 
 ## Why Poolifier?
 
-Poolifier is used to perform heavy CPU bound tasks on nodejs servers, it implements thread pools (yes, more thread pool implementations, so you can choose which one fit better for you) using [worker-threads](https://nodejs.org/api/worker_threads.html#worker_threads_worker_threads).  
```

Comment:
I expect only  threads to be used for CPU bound tasks, cluster workers should be used more for I/O intensive.
I expect thread pools to be used within the cluster worker when there are mixed CPU and I/O intensive tasks.

We surely need to benchmark those expectations